test/collection.js

```diff
@@ -506,7 +506,7 @@
     };
     collection.url = '/test';
     collection.fetch();
-    this.syncArgs.options.success();
+    this.syncArgs.options.success([]);
```

Comment:
Ok but that's a breaking change. Any way to still allow `success()` with no args?